.travis.yml

```
@@ -17,4 +17,13 @@ notifications:
     recipients:
       - ajax@bynder.com
     on_success: never
-    on_failure: always
\ No newline at end of file
+    on_failure: always
+
+deploy:
+  provider: pypi
+  username: __token__
+  password:
+    secure: $PYPI_TOKEN
+  on:
```

I think we also need to specify to only release on a certain python version, now it's trying on both 3.5 and 3.6 which results in a failure for 3.6 since it's uploading the same file again.

```suggestion
  on:
    python: "3.6"  # or the highest if we update the python list
```

---

Yes I noticed that in the builds. I am going to make it point to only 3.6 for now.